content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
 section: security
 ---
 
+The Jenkins project takes security seriously.
+We make every possible effort to ensure users can adequately secure their automation infrastructure.
```

Comment:
line break for new sentence: approve.
line break for logical section: sure. 
line break before "link": this too please. 
tab for next line:  wut? 

I guess I don't use a real text editor.  

And even if I did, I look at and comment on things in GitHub, meaning long lines produce multiple comments per line.